variables.tf

```diff
@@ -333,6 +333,26 @@ variable "enable_host_encryption" {
   default     = false
 }
 
+variable "private_dns_zone_id" {
+  description = <<EOT
+(Optional) Either the ID of Private DNS Zone which should be delegated to this Cluster,
+or System to have AKS manage this or None.
```

Comment:
Could we quote "System" and "None" here with `?

As the [provider's document](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/website/docs/r/kubernetes_cluster.html.markdown?plain=1#L156) shows, the description should be:

```text
(Optional) Either the ID of Private DNS Zone which should be delegated to this Cluster, `System` to have AKS manage this or `None`. In case of `None` you will need to bring your own DNS server and set up resolving, otherwise cluster will have issues after provisioning. Changing this forces a new resource to be created.
```

Reply:
@lonegunmanb done ,Please review